plugins/source/aws/codegen/recipes/cloudhsmv2.go

```diff
@@ -0,0 +1,47 @@
+package recipes
+
+import (
+	"github.com/aws/aws-sdk-go-v2/service/cloudhsmv2/types"
+	"github.com/cloudquery/plugin-sdk/codegen"
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func CloudHSMV2() []*Resource {
+	resources := []*Resource{
+		{
+			SubService: "clusters",
+			Struct:     &types.Cluster{},
+			Multiplex:  `client.ServiceAccountRegionMultiplexer("cloudhsmv2")`,
+			ExtraColumns: append(
+				defaultRegionalColumns,
+				[]codegen.ColumnDefinition{
+					{
+						Name:     "arn",
+						Type:     schema.TypeString,
+						Resolver: `resolveClusterArn`,
+						Options:  schema.ColumnCreationOptions{PrimaryKey: true},
+					},
```

Comment:
For consistency with other AWS resources, let's add a `ResolveTagField` resolver for `tags` here (using field `TagList`)